src/garage/tf/core/parameter.py

```diff
@@ -1,7 +1,6 @@
 """Parameter layer in TensorFlow."""
 
 import tensorflow as tf
-from tensorflow.python.ops.gen_array_ops import broadcast_to
```

Comment:
Why is this file in this change? Please undo the changes to it (`git checkout origin/master src/garage/tf/core/parameter.py`).